### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by rhigman** [thoth-pub/thoth#322]
Users of Bulma have noted that it has a number of accessibility issues (https://github.com/jgthms/bulma/issues/53) and the creator acknowledges this (https://bulma.io/alternative-to-bootstrap/). Our use of clickable table headers/rows, buttons within tables, etc is also a problem as these cannot be navigated to using only the keyboard. We should review Thoth's compliance with the WCAG and determine how to make it open to all.
